### PR TITLE
Solution promises_practise

### PR DESCRIPTION
**Pull request by NazariiAlieksieiev**
[DEMO LINK](https://NazariiAlieksieiev.github.io/js_promises_practice_DOM/)

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1 +1,62 @@
...
+  });
+});
+
+const thirdPromise = new Promise((resolve) => {
+  let left = 0;
+  let right = 0;
+
+  document.addEventListener('mousedown', (e) => {
+    left += e.button;
+    right += e.button;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/scripts/main.js:29]: wouldn't simple `=` work?

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
Amazing job, but please provide DEMO_LINK so mentor would check your work

File: src/scripts/main.js (hunk 1 above)
Comment (NazariiAlieksieiev) [reply src/scripts/main.js:29]: I Fixed It)

**Review by Volodymyr-Melnyk-Fron10der — APPROVED** [review]
ClappingLeonardoDicaprioGIF

**Pull request closed without merging**